### ISSUE TITLE
Sync reports are not nicely formatted

### ISSUE THREAD

**Issue by jacobperron** [ros-infrastructure/reprepro-updater#87]
Since a recent update, we see a literal byte string for the report instead of the nicely formatted markdown that used to be there.

[For example](http://build.ros2.org/job/Frel_sync-packages-to-main/11/):

```
b'## Package Updates for foxy\n\n### Added Packages [41]:\n\n * ros-foxy-grbl-msgs: 0.0.2-2\n * ros-foxy-grbl-msgs-dbgsym: 0.0.2-2\n * ros-foxy-joy-teleop: 1.2.1-1\n * ros-foxy-key-teleop: 1.2.1-1\n * [ros-foxy-mapviz](https://github.com/swri-robotics/mapviz): 2.1.0-1\n * ros-foxy-mapviz-dbgsym: 2.1.0-1\n * [ros-foxy-mapviz-interfaces](https://github.com/swri-robotics/mapviz): 2.1.0-1\n * ros-foxy-mapviz-interfaces-dbgsym: 2.1.0-1\n * [ros-foxy-mapviz-plugins](https://github.com/swri-robotics/mapviz): 2.1.0-1\n * ros-foxy-mapviz-plugins-dbgsym: 2.1.0-1\n * ros-foxy-mouse-teleop: 1.2.1-1\n * [ros-foxy-multires-image](https://github.com/swri-robotics/mapviz): 2.1.0-1\n * ros-foxy-multires-image-dbgsym: 2.1.0-1\n * [ros-foxy-osqp-vendor](https://github.com/oxfordcontrol/osqp): 0.0.2-1\n * ros-foxy-osqp-vendor-dbgsym: 0.0.2-1\n * ros-foxy-rc-common-msgs: 0.5.3-1\n * ros-foxy-rc-common-msgs-dbgsym: 0.5.3-1\n * [ros-foxy-sick-scan2](http://wiki.ros.org/sick_scan2): 0.1.9-1\n * ros-foxy-sick-scan2-dbgsym: 0.1.9-1\n * ros-foxy-teleop-tools: 1.2.1-1\n * ros-foxy-teleop-tools-msgs: 1.2.1-1\n * ros-foxy-teleop-tools-msgs-dbgsym: 1.2.1-1\n * [ros-foxy-tile-map](https://github.com/swri-robotics/mapviz): 2.1.0-1\n * ros-foxy-tile-map-dbgsym: 2.1.0-1\n * [ros-foxy-turtlebot3](http://turtlebot3.robotis.com): 2.1.0-1\n * [ros-foxy-turtlebot3-bringup](http://turtlebot3.robotis.com): 2.1.0-1\n * [ros-foxy-turtlebot3-cartographer](http://turtlebot3.robotis.com): 2.1.0-1\n * [ros-foxy-turtlebot3-description](http://turtlebot3.robotis.com): 2.1.0-1\n * [ros-foxy-turtlebot3-example](http://turtlebot3.robotis.com): 2.1.0-1\n * [ros-foxy-turtlebot3-navigation2](http://turtlebot3.robotis.com): 2.1.0-1\n * [ros-foxy-turtlebot3-node](http://turtlebot3.robotis.com): 2.1.0-1\n * ros-foxy-turtlebot3-node-dbgsym: 2.1.0-1\n * [ros-foxy-turtlebot3-teleop](http://turtlebot3.robotis.com): 2.1.0-1\n * [ros-foxy-ublox](http://wiki.ros.org/ublox): 2.0.0-1\n * [ros-foxy-ublox-gps](http://ros.org/wiki/ublox): 2.0.0-1\n * ros-foxy-ublox-gps-dbgsym: 2.0.0-1\n * [ros-foxy-ublox-msgs](http://ros.org/wiki/ublox): 2.0.0-1\n * ros-foxy-ublox-msgs-dbgsym: 2.0.0-1\n * [ros-foxy-ublox-serialization](http://ros.org/wiki/ublox): 2.0.0-1\n * ros-foxy-udp-msgs: 0.0.2-1\n * ros-foxy-udp-msgs-dbgsym: 0.0.2-1\n\n### Updated Packages [79]:\n\n * [ros-foxy-cartographer-ros](https://github.com/ros2/cartographer_ros): 1.0.9001-1 -> 1.0.9002-1\n * ros-foxy-cartographer-ros-dbgsym: 1.0.9001-1 -> 1.0.9002-1\n * [ros-foxy-cartographer-ros-msgs](https://github.com/ros2/cartographer_ros): 1.0.9001-1 -> 1.0.9002-1\n * ros-foxy-cartographer-ros-msgs-dbgsym: 1.0.9001-1 -> 1.0.9002-1\n * [ros-foxy-contracts-lite-vendor](https://github.com/ros-safety/contracts_lite_vendor): 0.4.0-1 -> 0.4.1-1\n * ros-foxy-examples-tf2-py: 0.13.5-1 -> 0.13.6-1\n * [ros-foxy-geometry2](http://www.ros.org/wiki/geometry2): 0.13.5-1 -> 0.13.6-1\n * ros-foxy-grbl-ros: 0.2.1-1 -> 0.0.15-1\n * ros-foxy-launch-ros: 0.10.3-1 -> 0.11.0-1\n * ros-foxy-launch-testing-ros: 0.10.3-1 -> 0.11.0-1\n * ros-foxy-lgsvl-bridge: 0.1.3-1 -> 0.2.1-1\n * ros-foxy-lgsvl-bridge-dbgsym: 0.1.3-1 -> 0.2.1-1\n * ros-foxy-lgsvl-msgs: 0.0.3-1 -> 0.0.4-1\n * ros-foxy-lgsvl-msgs-dbgsym: 0.0.3-1 -> 0.0.4-1\n * [ros-foxy-mavlink](https://mavlink.io/en/): 2020.9.10-1 -> 2020.10.11-1\n * [ros-foxy-mrpt2](https://www.mrpt.org/): 2.1.0-3 -> 2.1.3-1\n * ros-foxy-mrpt2-dbgsym: 2.1.0-3 -> 2.1.3-1\n * ros-foxy-performance-test-fixture: 0.0.4-1 -> 0.0.5-1\n * ros-foxy-performance-test-fixture-dbgsym: 0.0.4-1 -> 0.0.5-1\n * [ros-foxy-rc-genicam-api](http://wiki.ros.org/rc_genicam_api): 2.4.1-1 -> 2.4.4-1\n * ros-foxy-rc-genicam-api-dbgsym: 2.4.1-1 -> 2.4.4-1\n * ros-foxy-rcl: 1.1.8-1 -> 1.1.9-1\n * ros-foxy-rcl-action: 1.1.8-1 -> 1.1.9-1\n * ros-foxy-rcl-action-dbgsym: 1.1.8-1 -> 1.1.9-1\n * ros-foxy-rcl-dbgsym: 1.1.8-1 -> 1.1.9-1\n * ros-foxy-rcl-lifecycle: 1.1.8-1 -> 1.1.9-1\n * ros-foxy-rcl-lifecycle-dbgsym: 1.1.8-1 -> 1.1.9-1\n * ros-foxy-rcl-yaml-param-parser: 1.1.8-1 -> 1.1.9-1\n * ros-foxy-rcl-yaml-param-parser-dbgsym: 1.1.8-1 -> 1.1.9-1\n * ros-foxy-rcpputils: 1.3.0-1 -> 1.3.1-1\n * ros-foxy-rcpputils-dbgsym: 1.3.0-1 -> 1.3.1-1\n * ros-foxy-rmw: 1.0.1-1 -> 1.0.2-1\n * ros-foxy-rmw-connext-cpp: 1.0.1-1 -> 1.0.2-1\n * ros-foxy-rmw-connext-cpp-dbgsym: 1.0.1-1 -> 1.0.2-1\n * ros-foxy-rmw-connext-shared-cpp: 1.0.1-1 -> 1.0.2-1\n * ros-foxy-rmw-connext-shared-cpp-dbgsym: 1.0.1-1 -> 1.0.2-1\n * ros-foxy-rmw-cyclonedds-cpp: 0.7.4-1 -> 0.7.5-1\n * ros-foxy-rmw-cyclonedds-cpp-dbgsym: 0.7.4-1 -> 0.7.5-1\n * ros-foxy-rmw-dbgsym: 1.0.1-1 -> 1.0.2-1\n * ros-foxy-rmw-fastrtps-cpp: 1.2.1-1 -> 1.2.2-1\n * ros-foxy-rmw-fastrtps-cpp-dbgsym: 1.2.1-1 -> 1.2.2-1\n * ros-foxy-rmw-fastrtps-dynamic-cpp: 1.2.1-1 -> 1.2.2-1\n * ros-foxy-rmw-fastrtps-dynamic-cpp-dbgsym: 1.2.1-1 -> 1.2.2-1\n * ros-foxy-rmw-fastrtps-shared-cpp: 1.2.1-1 -> 1.2.2-1\n * ros-foxy-rmw-fastrtps-shared-cpp-dbgsym: 1.2.1-1 -> 1.2.2-1\n * ros-foxy-rmw-implementation: 1.0.0-1 -> 1.0.1-1\n * ros-foxy-rmw-implementation-cmake: 1.0.1-1 -> 1.0.2-1\n * ros-foxy-rmw-implementation-dbgsym: 1.0.0-1 -> 1.0.1-1\n * ros-foxy-ros2launch: 0.10.3-1 -> 0.11.0-1\n * ros-foxy-system-modes: 0.4.0-1 -> 0.4.1-1\n * ros-foxy-system-modes-dbgsym: 0.4.0-1 -> 0.4.1-1\n * ros-foxy-system-modes-examples: 0.4.0-1 -> 0.4.1-1\n * ros-foxy-system-modes-examples-dbgsym: 0.4.0-1 -> 0.4.1-1\n * [ros-foxy-tf2](http://www.ros.org/wiki/tf2): 0.13.5-1 -> 0.13.6-1\n * [ros-foxy-tf2-bullet](http://www.ros.org/wiki/tf2_bullet): 0.13.5-1 -> 0.13.6-1\n * ros-foxy-tf2-dbgsym: 0.13.5-1 -> 0.13.6-1\n * ros-foxy-tf2-eigen: 0.13.5-1 -> 0.13.6-1\n * [ros-foxy-tf2-geometry-msgs](http://www.ros.org/wiki/tf2_ros): 0.13.5-1 -> 0.13.6-1\n * [ros-foxy-tf2-kdl](http://ros.org/wiki/tf2): 0.13.5-1 -> 0.13.6-1\n * [ros-foxy-tf2-msgs](http://www.ros.org/wiki/tf2_msgs): 0.13.5-1 -> 0.13.6-1\n * ros-foxy-tf2-msgs-dbgsym: 0.13.5-1 -> 0.13.6-1\n * [ros-foxy-tf2-py](http://ros.org/wiki/tf2_py): 0.13.5-1 -> 0.13.6-1\n * ros-foxy-tf2-py-dbgsym: 0.13.5-1 -> 0.13.6-1\n * [ros-foxy-tf2-ros](http://www.ros.org/wiki/tf2_ros): 0.13.5-1 -> 0.13.6-1\n * ros-foxy-tf2-ros-dbgsym: 0.13.5-1 -> 0.13.6-1\n * [ros-foxy-tf2-sensor-msgs](http://www.ros.org/wiki/tf2_ros): 0.13.5-1 -> 0.13.6-1\n * [ros-foxy-tf2-tools](http://www.ros.org/wiki/tf2_tools): 0.13.5-1 -> 0.13.6-1\n * [ros-foxy-webots-ros2](http://wiki.ros.org/webots_ros2): 1.0.1-2 -> 1.0.2-1\n * [ros-foxy-webots-ros2-abb](http://wiki.ros.org/webots_ros2): 1.0.1-2 -> 1.0.2-1\n * [ros-foxy-webots-ros2-core](http://wiki.ros.org/webots_ros2): 1.0.1-2 -> 1.0.2-1\n * [ros-foxy-webots-ros2-demos](http://wiki.ros.org/webots_ros2): 1.0.1-2 -> 1.0.2-1\n * [ros-foxy-webots-ros2-epuck](http://wiki.ros.org/webots_ros2): 1.0.1-2 -> 1.0.2-1\n * [ros-foxy-webots-ros2-examples](http://wiki.ros.org/webots_ros2): 1.0.1-2 -> 1.0.2-1\n * [ros-foxy-webots-ros2-importer](http://wiki.ros.org/webots_ros2): 1.0.1-2 -> 1.0.2-1\n * [ros-foxy-webots-ros2-msgs](http://wiki.ros.org/webots_ros2): 1.0.1-2 -> 1.0.2-1\n * ros-foxy-webots-ros2-msgs-dbgsym: 1.0.1-2 -> 1.0.2-1\n * [ros-foxy-webots-ros2-tiago](http://wiki.ros.org/webots_ros2): 1.0.1-2 -> 1.0.2-1\n * [ros-foxy-webots-ros2-universal-robot](http://wiki.ros.org/webots_ros2): 1.0.1-2 -> 1.0.2-1\n * [ros-foxy-webots-ros2-ur-e-description](http://wiki.ros.org/webots_ros2): 1.0.1-2 -> 1.0.2-1\n\n### Removed Packages [0]:\n\n\nThanks to all ROS maintainers who make packages available to the ROS community. The above list of packages was made possible by the work of the following maintainers: \n\n * Alejandro Hernandez Cordero\n * Anup Pemmaiah\n * Arne Nordmann\n * Bence Magyar\n * Chris Lalancette\n * Cyberbotics\n * Dirk Thomas\n * Emerson Knapp\n * Enrique Fernandez\n * Erik Boasson\n * Esteve Fernandez\n * Evan Flynn\n * Felix Ruess\n * Gareth Cross\n * Hadi Tabatabaee\n * Jacob Perron\n * Jeffrey Kane Johnson\n * Jose-Luis Blanco-Claraco\n * Karsten Knese\n * Martins Mozeiko\n * Michael Lehning\n * P. J. Reed\n * Veronica Lane\n * Vladimir Ermakov\n * Will Son\n * William Woodall\n'
```

As a workaround, we can find-and-replace literal `\n` strings with newlines and remove the `b'` prefix to get a nicely formatted report:

```
## Package Updates for foxy

### Added Packages [41]:

 * ros-foxy-grbl-msgs: 0.0.2-2
 * ros-foxy-grbl-msgs-dbgsym: 0.0.2-2
 * ros-foxy-joy-teleop: 1.2.1-1
 * ros-foxy-key-teleop: 1.2.1-1
 * [ros-foxy-mapviz](https://github.com/swri-robotics/mapviz): 2.1.0-1
 * ros-foxy-mapviz-dbgsym: 2.1.0-1
 * [ros-foxy-mapviz-interfaces](https://github.com/swri-robotics/mapviz): 2.1.0-1
 * ros-foxy-mapviz-interfaces-dbgsym: 2.1.0-1
 * [ros-foxy-mapviz-plugins](https://github.com/swri-robotics/mapviz): 2.1.0-1
 * ros-foxy-mapviz-plugins-dbgsym: 2.1.0-1
 * ros-foxy-mouse-teleop: 1.2.1-1
 * [ros-foxy-multires-image](https://github.com/swri-robotics/mapviz): 2.1.0-1
 * ros-foxy-multires-image-dbgsym: 2.1.0-1
 * [ros-foxy-osqp-vendor](https://github.com/oxfordcontrol/osqp): 0.0.2-1
 * ros-foxy-osqp-vendor-dbgsym: 0.0.2-1
 * ros-foxy-rc-common-msgs: 0.5.3-1
 * ros-foxy-rc-common-msgs-dbgsym: 0.5.3-1
 * [ros-foxy-sick-scan2](http://wiki.ros.org/sick_scan2): 0.1.9-1
 * ros-foxy-sick-scan2-dbgsym: 0.1.9-1
 * ros-foxy-teleop-tools: 1.2.1-1
 * ros-foxy-teleop-tools-msgs: 1.2.1-1
 * ros-foxy-teleop-tools-msgs-dbgsym: 1.2.1-1
 * [ros-foxy-tile-map](https://github.com/swri-robotics/mapviz): 2.1.0-1
 * ros-foxy-tile-map-dbgsym: 2.1.0-1
 * [ros-foxy-turtlebot3](http://turtlebot3.robotis.com): 2.1.0-1
 * [ros-foxy-turtlebot3-bringup](http://turtlebot3.robotis.com): 2.1.0-1
 * [ros-foxy-turtlebot3-cartographer](http://turtlebot3.robotis.com): 2.1.0-1
 * [ros-foxy-turtlebot3-description](http://turtlebot3.robotis.com): 2.1.0-1
 * [ros-foxy-turtlebot3-example](http://turtlebot3.robotis.com): 2.1.0-1
 * [ros-foxy-turtlebot3-navigation2](http://turtlebot3.robotis.com): 2.1.0-1
 * [ros-foxy-turtlebot3-node](http://turtlebot3.robotis.com): 2.1.0-1
 * ros-foxy-turtlebot3-node-dbgsym: 2.1.0-1
 * [ros-foxy-turtlebot3-teleop](http://turtlebot3.robotis.com): 2.1.0-1
 * [ros-foxy-ublox](http://wiki.ros.org/ublox): 2.0.0-1
 * [ros-foxy-ublox-gps](http://ros.org/wiki/ublox): 2.0.0-1
 * ros-foxy-ublox-gps-dbgsym: 2.0.0-1
 * [ros-foxy-ublox-msgs](http://ros.org/wiki/ublox): 2.0.0-1
 * ros-foxy-ublox-msgs-dbgsym: 2.0.0-1
 * [ros-foxy-ublox-serialization](http://ros.org/wiki/ublox): 2.0.0-1
 * ros-foxy-udp-msgs: 0.0.2-1
 * ros-foxy-udp-msgs-dbgsym: 0.0.2-1

### Updated Packages [79]:

 * [ros-foxy-cartographer-ros](https://github.com/ros2/cartographer_ros): 1.0.9001-1 -> 1.0.9002-1
 * ros-foxy-cartographer-ros-dbgsym: 1.0.9001-1 -> 1.0.9002-1
 * [ros-foxy-cartographer-ros-msgs](https://github.com/ros2/cartographer_ros): 1.0.9001-1 -> 1.0.9002-1
 * ros-foxy-cartographer-ros-msgs-dbgsym: 1.0.9001-1 -> 1.0.9002-1
 * [ros-foxy-contracts-lite-vendor](https://github.com/ros-safety/contracts_lite_vendor): 0.4.0-1 -> 0.4.1-1
 * ros-foxy-examples-tf2-py: 0.13.5-1 -> 0.13.6-1
 * [ros-foxy-geometry2](http://www.ros.org/wiki/geometry2): 0.13.5-1 -> 0.13.6-1
 * ros-foxy-grbl-ros: 0.2.1-1 -> 0.0.15-1
 * ros-foxy-launch-ros: 0.10.3-1 -> 0.11.0-1
 * ros-foxy-launch-testing-ros: 0.10.3-1 -> 0.11.0-1
 * ros-foxy-lgsvl-bridge: 0.1.3-1 -> 0.2.1-1
 * ros-foxy-lgsvl-bridge-dbgsym: 0.1.3-1 -> 0.2.1-1
 * ros-foxy-lgsvl-msgs: 0.0.3-1 -> 0.0.4-1
 * ros-foxy-lgsvl-msgs-dbgsym: 0.0.3-1 -> 0.0.4-1
 * [ros-foxy-mavlink](https://mavlink.io/en/): 2020.9.10-1 -> 2020.10.11-1
 * [ros-foxy-mrpt2](https://www.mrpt.org/): 2.1.0-3 -> 2.1.3-1
 * ros-foxy-mrpt2-dbgsym: 2.1.0-3 -> 2.1.3-1
 * ros-foxy-performance-test-fixture: 0.0.4-1 -> 0.0.5-1
 * ros-foxy-performance-test-fixture-dbgsym: 0.0.4-1 -> 0.0.5-1
 * [ros-foxy-rc-genicam-api](http://wiki.ros.org/rc_genicam_api): 2.4.1-1 -> 2.4.4-1
 * ros-foxy-rc-genicam-api-dbgsym: 2.4.1-1 -> 2.4.4-1
 * ros-foxy-rcl: 1.1.8-1 -> 1.1.9-1
 * ros-foxy-rcl-action: 1.1.8-1 -> 1.1.9-1
 * ros-foxy-rcl-action-dbgsym: 1.1.8-1 -> 1.1.9-1
 * ros-foxy-rcl-dbgsym: 1.1.8-1 -> 1.1.9-1
 * ros-foxy-rcl-lifecycle: 1.1.8-1 -> 1.1.9-1
 * ros-foxy-rcl-lifecycle-dbgsym: 1.1.8-1 -> 1.1.9-1
 * ros-foxy-rcl-yaml-param-parser: 1.1.8-1 -> 1.1.9-1
 * ros-foxy-rcl-yaml-param-parser-dbgsym: 1.1.8-1 -> 1.1.9-1
 * ros-foxy-rcpputils: 1.3.0-1 -> 1.3.1-1
 * ros-foxy-rcpputils-dbgsym: 1.3.0-1 -> 1.3.1-1
 * ros-foxy-rmw: 1.0.1-1 -> 1.0.2-1
 * ros-foxy-rmw-connext-cpp: 1.0.1-1 -> 1.0.2-1
 * ros-foxy-rmw-connext-cpp-dbgsym: 1.0.1-1 -> 1.0.2-1
 * ros-foxy-rmw-connext-shared-cpp: 1.0.1-1 -> 1.0.2-1
 * ros-foxy-rmw-connext-shared-cpp-dbgsym: 1.0.1-1 -> 1.0.2-1
 * ros-foxy-rmw-cyclonedds-cpp: 0.7.4-1 -> 0.7.5-1
 * ros-foxy-rmw-cyclonedds-cpp-dbgsym: 0.7.4-1 -> 0.7.5-1
 * ros-foxy-rmw-dbgsym: 1.0.1-1 -> 1.0.2-1
 * ros-foxy-rmw-fastrtps-cpp: 1.2.1-1 -> 1.2.2-1
 * ros-foxy-rmw-fastrtps-cpp-dbgsym: 1.2.1-1 -> 1.2.2-1
 * ros-foxy-rmw-fastrtps-dynamic-cpp: 1.2.1-1 -> 1.2.2-1
 * ros-foxy-rmw-fastrtps-dynamic-cpp-dbgsym: 1.2.1-1 -> 1.2.2-1
 * ros-foxy-rmw-fastrtps-shared-cpp: 1.2.1-1 -> 1.2.2-1
 * ros-foxy-rmw-fastrtps-shared-cpp-dbgsym: 1.2.1-1 -> 1.2.2-1
 * ros-foxy-rmw-implementation: 1.0.0-1 -> 1.0.1-1
 * ros-foxy-rmw-implementation-cmake: 1.0.1-1 -> 1.0.2-1
 * ros-foxy-rmw-implementation-dbgsym: 1.0.0-1 -> 1.0.1-1
 * ros-foxy-ros2launch: 0.10.3-1 -> 0.11.0-1
 * ros-foxy-system-modes: 0.4.0-1 -> 0.4.1-1
 * ros-foxy-system-modes-dbgsym: 0.4.0-1 -> 0.4.1-1
 * ros-foxy-system-modes-examples: 0.4.0-1 -> 0.4.1-1
 * ros-foxy-system-modes-examples-dbgsym: 0.4.0-1 -> 0.4.1-1
 * [ros-foxy-tf2](http://www.ros.org/wiki/tf2): 0.13.5-1 -> 0.13.6-1
 * [ros-foxy-tf2-bullet](http://www.ros.org/wiki/tf2_bullet): 0.13.5-1 -> 0.13.6-1
 * ros-foxy-tf2-dbgsym: 0.13.5-1 -> 0.13.6-1
 * ros-foxy-tf2-eigen: 0.13.5-1 -> 0.13.6-1
 * [ros-foxy-tf2-geometry-msgs](http://www.ros.org/wiki/tf2_ros): 0.13.5-1 -> 0.13.6-1
 * [ros-foxy-tf2-kdl](http://ros.org/wiki/tf2): 0.13.5-1 -> 0.13.6-1
 * [ros-foxy-tf2-msgs](http://www.ros.org/wiki/tf2_msgs): 0.13.5-1 -> 0.13.6-1
 * ros-foxy-tf2-msgs-dbgsym: 0.13.5-1 -> 0.13.6-1
 * [ros-foxy-tf2-py](http://ros.org/wiki/tf2_py): 0.13.5-1 -> 0.13.6-1
 * ros-foxy-tf2-py-dbgsym: 0.13.5-1 -> 0.13.6-1
 * [ros-foxy-tf2-ros](http://www.ros.org/wiki/tf2_ros): 0.13.5-1 -> 0.13.6-1
 * ros-foxy-tf2-ros-dbgsym: 0.13.5-1 -> 0.13.6-1
 * [ros-foxy-tf2-sensor-msgs](http://www.ros.org/wiki/tf2_ros): 0.13.5-1 -> 0.13.6-1
 * [ros-foxy-tf2-tools](http://www.ros.org/wiki/tf2_tools): 0.13.5-1 -> 0.13.6-1
 * [ros-foxy-webots-ros2](http://wiki.ros.org/webots_ros2): 1.0.1-2 -> 1.0.2-1
 * [ros-foxy-webots-ros2-abb](http://wiki.ros.org/webots_ros2): 1.0.1-2 -> 1.0.2-1
 * [ros-foxy-webots-ros2-core](http://wiki.ros.org/webots_ros2): 1.0.1-2 -> 1.0.2-1
 * [ros-foxy-webots-ros2-demos](http://wiki.ros.org/webots_ros2): 1.0.1-2 -> 1.0.2-1
 * [ros-foxy-webots-ros2-epuck](http://wiki.ros.org/webots_ros2): 1.0.1-2 -> 1.0.2-1
 * [ros-foxy-webots-ros2-examples](http://wiki.ros.org/webots_ros2): 1.0.1-2 -> 1.0.2-1
 * [ros-foxy-webots-ros2-importer](http://wiki.ros.org/webots_ros2): 1.0.1-2 -> 1.0.2-1
 * [ros-foxy-webots-ros2-msgs](http://wiki.ros.org/webots_ros2): 1.0.1-2 -> 1.0.2-1
 * ros-foxy-webots-ros2-msgs-dbgsym: 1.0.1-2 -> 1.0.2-1
 * [ros-foxy-webots-ros2-tiago](http://wiki.ros.org/webots_ros2): 1.0.1-2 -> 1.0.2-1
 * [ros-foxy-webots-ros2-universal-robot](http://wiki.ros.org/webots_ros2): 1.0.1-2 -> 1.0.2-1
 * [ros-foxy-webots-ros2-ur-e-description](http://wiki.ros.org/webots_ros2): 1.0.1-2 -> 1.0.2-1

### Removed Packages [0]:


Thanks to all ROS maintainers who make packages available to the ROS community. The above list of packages was made possible by the work of the following maintainers: 

 * Alejandro Hernandez Cordero
 * Anup Pemmaiah
 * Arne Nordmann
 * Bence Magyar
 * Chris Lalancette
 * Cyberbotics
 * Dirk Thomas
 * Emerson Knapp
 * Enrique Fernandez
 * Erik Boasson
 * Esteve Fernandez
 * Evan Flynn
 * Felix Ruess
 * Gareth Cross
 * Hadi Tabatabaee
 * Jacob Perron
 * Jeffrey Kane Johnson
 * Jose-Luis Blanco-Claraco
 * Karsten Knese
 * Martins Mozeiko
 * Michael Lehning
 * P. J. Reed
 * Veronica Lane
 * Vladimir Ermakov
 * Will Son
 * William Woodall

```

**Comment by cottsay** [comment]
This may be related to ros-infrastructure/ros_buildfarm#842, but there were other changes made here to support that change (like #84)

**Comment by nuclearsandwich** [comment]
This was resolved by #88 but because the active source is on the `refactor` branch while the active configs are on master the issue wasn't auto-closed.
